src/graph/types.ts

```diff
@@ -22,6 +22,13 @@ export type IDynamicPerson = (MicrosoftGraph.User | MicrosoftGraph.Person | Micr
    * @type {string}
    */
   personImage?: string;
+
+  /**
+   * personDetails.email is a toolkit injected property to manually set a singular email address for a user.
+   *
+   * @type {string}
+   */
+  email?: string;
```

Comment:
I'm not sure what value this brings? It doesn't seem necessary as all the different types already have some kind of field for email

Reply:
I wasn't sure if the `User.mail` prop could be used for something else. Sounds good, I've updated:

462e6dc